index.js

```diff
@@ -203,7 +207,9 @@ class CoverageSonar extends CoverageBase {
         return getCoveragePercentage({ jobId, startTime, endTime })
             .then(coveragePercentage => ({
                 coverage: coveragePercentage,
-                projectUrl
+                projectUrl,
+                sdApiAuthUrl,
```

Comment:
Adding it to this function may not work since when we try to call this inside the launcher, `getCoveragePercentage` will probably fail because there is no such project yet

Reply:
Overloading the getInfo method so it'll just return the basic info when startTime and endTime are not passed in.